DESCRIPTION

```diff
@@ -42,6 +42,7 @@ Imports:
     rlang,
     tidyr,
     tidyselect,
+    withr,
```

Comment:
Do we need to depend on a package that is used only in a sourcing file in `data-raw`?

Reply:
No, thanks!!